dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -300,6 +300,18 @@
             <column name="topicai" type="varchar(150 BYTE)"/>
         </addColumn>
     </changeSet>
+    <changeSet id="lambdaEntryNameConvertEmptyStringToNull" author="dyuen">
+        <update schemaName="public"
+                tableName="lambdaevent">
+            <column name="entryname"/>
+            <where>entryname = ''</where>
+        </update>
+    </changeSet>
+    <changeSet id="lambdaEntryNameNotEmptyConstraint" author="dyuen" >
+        <sql dbms="postgresql">
```

Comment:
Although it's extremely unlikely to happen, especially since 1.15 calls for 1.14 downtime, shouldn't this be in the same changeset as line 303? In theory an empty workflow could get introduced between the two transactions.